gcsio/src/main/java/com/google/cloud/hadoop/gcsio/GoogleCloudStorageImpl.java

```diff
@@ -371,6 +379,14 @@ public GoogleCloudStorageImpl(
 
     this.httpRequestInitializer = this.storage.getRequestFactory().getInitializer();
 
+    if (MetricsSink.CLOUD_MONITORING.equals(this.storageOptions.getMetricsSink())) {
+      Credentials credentials = ((RetryHttpInitializer) httpRequestInitializer).getCredentials();
```

Comment:
Seems like it worth inlining this var to unblock usage of ternary operator instead of `if`